lib/que/formatters/output_formatter.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'active_support/logger'
+
+module Que
+  module Formatters
+    class OutputFormatter < ::Logger::Formatter
```

Comment:
Based on my comments on the other PR: https://github.com/homestars/accounts-management-api/pull/112#pullrequestreview-638441711, these changes may not be needed. 🤞